File: android_taskdemo/app/src/main/java/top/zcwfeng/taskdemo/MainActivity.java
package top.zcwfeng.taskdemo;

import android.os.Bundle;

public class MainActivity extends BaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected String getTAG(){
        return "MainActivity";
    }
}

File: android_hilt/daggersample/src/main/java/top/zcwfeng/daggersample/MainActivity.kt
package top.zcwfeng.daggersample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import top.zcwfeng.daggersample.data.User
import top.zcwfeng.daggersample.databinding.ActivityMainBinding
import javax.inject.Inject

class MainActivity : AppCompatActivity() {
    @Inject
    lateinit var user: User
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        (application as DaggerApp).appComponent.inject(this)
//        binding.userTextView.text = "${user.name}使用 Dagger 的心情是：${user.mood}"
        user.mood = "超级难受"

    }
}
